### ISSUE TITLE
No output when "git blame" is run. 

### ISSUE THREAD

**Issue by dtardoin** [frou/st3-gitblame#33]
When I run the "git blame" command there's no UI indication, but this exception is thrown in the console: 
`Git blame: Unexpected error: 'NoneType' object is not subscriptable`

Git Blame Version: v1.1.3
Sublime Version: 3.1.1
OS: Ubuntu 18.04

**Comment by peterkos** [comment]
I'm having the same issue on macOS 10.14.2, ST 3.1.1.

Interestingly, I only get this issue when trying to blame a single line. It works fine when I attempt to blame all lines in the file.

More error logs for reference: (edited slightly for clarity)
```
File "git-blame.py", line 170, in on_phantom_close
AttributeError: 'NoneType' object has no attribute 'decode'
Git blame: Unexpected error: 'NoneType' object is not subscriptable
```

**Comment by frou** [comment]
@dtardoin I played around and found that I could reproduce that when asking for blame on a file that had never been saved to disk (i.e. File > New File , then run the Blame command). Is that the scenario you had?

@peterkos It looks like the unix command [unexpand](https://linux.die.net/man/1/unexpand) is somehow involved in your git setup (maybe via a `.gitconfig` file somewhere?) The unmangled version of the text you're seeing in the UI is probably from e.g.

    $ unexpand --foo
    unexpand: illegal option -- -
    usage: unexpand [-a | -t tablist] [file ...]

**Comment by peterkos** [comment]
Yep, that was it! I left a spare configuration option that didn't quite work, and that was causing the issue. All good now!

**Comment by frou** [comment]
I can only assume that the original issue reported here was fixed by #26
